packages/cli/src/commands/worldgen.ts

```diff
@@ -1,17 +1,12 @@
+import path from "node:path";
 import type { CommandModule } from "yargs";
-import { loadConfig } from "@latticexyz/config/node";
+import { loadConfig, resolveConfigPath } from "@latticexyz/config/node";
 import { World as WorldConfig } from "@latticexyz/world";
 import { worldgen } from "@latticexyz/world/node";
-import { getSrcDirectory } from "@latticexyz/common/foundry";
-import path from "path";
-import { rmSync } from "fs";
-import { getExistingContracts } from "../utils/getExistingContracts";
 
 type Options = {
   configPath?: string;
   clean?: boolean;
-  srcDir?: string;
-  config?: WorldConfig;
```

Comment:
I don't think this option ever worked - it wasn't part of `builder` method and isn't possible to pass in an object to CLI anyway.